.github/workflows/multi-node.yml

```diff
@@ -0,0 +1,63 @@
+name: Multi node test
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags-ignore: [v.*]
+jobs:
+  run-multi-node-tests:
+    name: Multi Node Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix: # *DO NOT EDIT* the following line is generated by running ./scripts/gh-generate-projects-matrix.sh from the root folder
+        project: [akka-cluster, akka-cluster-metrics, akka-cluster-sharding, akka-cluster-sharding-typed, akka-cluster-tools, akka-cluster-typed, akka-distributed-data, akka-remote-tests]
+      fail-fast: true
```

Comment:
I love running them on parallel! +100

Reply:
Parallel testing s not going to work with the extra nodes, so not sure we should introduce it if we don't need it later on.

Reply:
At the moment, running tests in a matrix let's us have a clearer view of what's flaky and what isn't. 

If/when we move to using external nodes we'll re-assess if we can run tests in parallel or not. Maybe we go for dynamic micro instances for each of those jobs, or maybe we reuse-preexesitng VMs,... 

Right now we need fast feedback loops and I think running the multi-node test parallelizing by module will shorten the feedback loop. It's a hunch. This morning on the shower I had the exact same idea.


Actually, Lagom and Play (which use Travis CI) do run split some long jobs into smaller bits to run them in parallelt8. In that case it's `scripted` tests. Of course, there the advantage is twofold because in case of a flaky failure Travis supports re-running a single job.

Reply:
Here's a sample of a non-matrix test result with a failure: https://github.com/akka/akka/pull/30449/checks?check_run_id=3252611503

Good luck fishing out the FAILURE/s...

Reply:
TBQH it's pretty easy if you search for `***` usually ;)